gdnative-core/src/core_types/quat.rs

```diff
@@ -1,4 +1,7 @@
-use super::IsEqualApprox;
+use super::{Basis, IsEqualApprox, Vector3, CMP_EPSILON};
+use core::f32::consts::FRAC_PI_2;
```

Comment:
I would use `std`, not `core` (we anyway require the full standard library, `core` is mainly useful for embedded systems).